gp-templates/translations.php

```diff
@@ -75,19 +75,34 @@
 	<?php
 	$filter_links = array();
 
+	$filters_and_sort = array_merge( array_filter( $filters ), array_filter( $sort ) );
+
+	$all_filters = array(
+		'status' => 'current_or_waiting_or_fuzzy_or_untranslated',
+	);
+
+	array() === array_diff( $all_filters, $filters_and_sort ) || array() === $filters_and_sort ? $extra_classes = array( 'class' => 'filter-current' ) : $extra_classes = array();
```

Comment:
I updated the PR with a minor change as `array( 'class' => 'filter-current' )` breaks the coding standard so I created a variable to hold the array instead.